doc/arkode/guide/source/Usage/SplittingStep/Skeleton.rst

```diff
@@ -0,0 +1,115 @@
+.. ----------------------------------------------------------------
+   Programmer(s): Steven B. Roberts @ LLNL
+   ----------------------------------------------------------------
+   Based on MRIStep by David J. Gardner @ LLNL
+   Daniel R. Reynolds @ SMU
+   ----------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2024, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. _ARKODE.Usage.SplittingStep.Skeleton:
+
+A skeleton of the user's main program
+============================================
+
+While SplittingStep usage generally follows the same pattern as the rest of
+ARKODE, since it is the composition of other steppers, we summarize the
+differences in using SplittingStep here.  Steps that are unchanged from the
+skeleton program presented in :numref:`ARKODE.Usage.Skeleton` are *italicized*.
+
+.. index:: SplittingStep user main program
+
+#. *Initialize parallel or multi-threaded environment, if appropriate.*
+
+#. *Create the SUNDIALS simulation context object*
+
+#. *Set problem dimensions, etc.*
+
+#. *Set vector of initial values*
+
+#. Create a stepper object for each problem partition
+
+   * If using ARKStep as an inner integrator, create the ARKStep object with
+     :c:func:`ARKStepCreate` and configure the integrator as desired for
+     evolving the partition. See sections :numref:`ARKODE.Usage.Skeleton`,
+     :numref:`ARKODE.Usage.OptionalInputs`, and
+     :numref:`ARKODE.Usage.ARKStep.OptionalInputs` for details on configuring
+     ARKStep.
```

Comment:
Should this paragraph be so specific to ARKStep?  Perhaps a final sentence could be added with something like "We note that other ARKODE steppers may also be used as an inner integrator, using similar steps."